### ISSUE TITLE
Medium severity vulnerability found in lodash

### ISSUE THREAD

**Issue by dep** [caolan/async#1620]
`snyk` reports a Regular Expression Denial of Service vulnerability on one of your dependencies, lodash `4.17.5`.

```
✗ Medium severity vulnerability found in lodash
  Description: Regular Expression Denial of Service (ReDoS)
  Info: https://snyk.io/vuln/SNYK-JS-LODASH-73639
  Introduced through: snyk@1.89.0
  From: snyk@1.89.0 > lodash@4.17.5
  Remediation:
    Your dependencies are out of date, otherwise you would be using a newer version of lodash. 
    Try deleting node_modules, reinstalling and running `snyk test` again. If the problem persists, one of your dependencies may be bundling outdated modules.
```

and

```
Analyzing npm dependencies for package.json
Querying vulnerabilities database...
Tested 255 dependencies for known vulnerabilities, found 3 vulnerabilities, 23 vulnerable paths.

? 2 vulnerabilities introduced via async@2.6.1
- info: https://snyk.io/package/npm/async/2.6.1
```

Thanks in advance!

**Comment by dscalzi** [comment]
Bump

**Comment by aearly** [comment]
We don't use that method so it doesn't apply to us. We removed lodash in
v3.0. I think there were some issues for us to migrate off that minor
version of lodash on the 2.x line of Async.

**Comment by dscalzi** [comment]
I think a simple patch update would do the trick.

![image](https://user-images.githubusercontent.com/9703873/52532802-50c35180-2cf8-11e9-9144-484e0396a39e.png)

**Comment by aearly** [comment]
Updating lodash was no issue,  I've published v2.6.2 with the update.
